controllers/calendar.js

```diff
@@ -43,9 +44,11 @@ router.get('/events/', (req, res, next) => {
 
 router.post('/events/', (req, res, next) => {
 	// eslint-disable-next-line no-underscore-dangle
-	req.body.startDate = moment(req.body.startDate, 'DD.MM.YYYY HH:mm')._d.toLocalISOString();
+	req.body.startDate = timesHelper.dateTimeStringToMoment(req.body.startDate)
```

Comment:
why does req.body.startDate not already have the right format defined in the browser?